crates/bevy_math/src/primitives/dim3.rs

```diff
@@ -85,6 +91,13 @@ impl std::ops::Neg for Direction3d {
     }
 }
 
+impl std::ops::Mul<f32> for Direction3d {
+    type Output = Vec3;
+    fn mul(self, rhs: f32) -> Self::Output {
+        self.0 * rhs
+    }
+}
+
```

Comment:
This should be implemented for `Direction2d` as well for consistency. It should probably also be implemented the other way around, i.e. `impl Mul<Direction3d> for f32`.